assignments/arrays.js

```diff
@@ -63,32 +63,53 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`car is ${inventory[22]["car_year"]} `);
```

Comment:
You were asked for car 33, but you are looking at array position 22?  The solution is:

console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`)

it is 32 because all arrays start at 0, so car 33 is actually array position 32 (0-32)